lib/mi_bridges/driver/base_page.rb

```diff
@@ -78,6 +78,24 @@ def log(description, *text)
           logger.debug("#{description.upcase}: #{text.join(', ')}")
         end
       end
+
+      def fill_in_birthday_fields(member)
```

Comment:
Looks like this method requires `member.birthday` more than it needs the full `member` - what do you think about passing in that instead?

Reply:
I like it! done at a1ab73d